src/parse/ast/bound.ts

```diff
@@ -4,6 +4,15 @@ import type { NumberLiteral } from "../../utils/numericLiterals.ts"
 import type { inferAst, validateAst } from "./ast.ts"
 import type { astToString } from "./utils.ts"
 
+/**
+ * @operator {@link validateBound | bound}
```

Comment:
I realize I maybe should've added the feedback on the generated files here but I'll just skip adding it again.